spring-cloud-gcp-logging/src/test/java/com/google/cloud/spring/logging/LoggingAppenderTests.java

```diff
@@ -65,32 +62,14 @@ void testGetLoggingOptions() {
         .isNotNull()
         .contains("spring-cloud-gcp-logging")
         .contains("Spring");
-
-    loggingAppender.start();
-    // java-logging-logback appender default is ERROR
-    verify(logging).setFlushSeverity(com.google.cloud.logging.Severity.ERROR);
-  }
-
-  @Test
-  void testDisablingFlushLevel() {
-    LoggingAppender loggingAppender = new TestLoggingAppender();
-    loggingAppender.setLogDestinationProjectId("my-log-destination-project");
-    loggingAppender.setFlushLevel(Level.OFF);
-
-    loggingAppender.start();
-    verify(logging, never()).setFlushSeverity(any());
   }
 
   @Test
   void testSetLogDestinationProjectId() {
     LoggingAppender loggingAppender = new TestLoggingAppender();
     loggingAppender.setCredentialsFile("src/test/resources/fake-project-key.json");
     loggingAppender.setLogDestinationProjectId("my-log-destination-project");
-    assertThat(loggingAppender.getLoggingOptions().getCredentials()).isNotNull();
```

Comment:
Can I ask why removing this check? It seems to be a valid check to see if credentials file is respected by loggingAppender.

Reply:
This test seems overlap with the previous one.

Reply:
That's fair. I was not looking at that one.